recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -70,6 +70,8 @@ do_install_append () {
     install -m 0700 -d ${D}${libdir}/sota/conf.d
     install -m 0700 -d ${D}${sysconfdir}/sota/conf.d
 
+    echo "[uptane]\nprimary_ecu_hardware_id = ${HARDWARE_ID}\n" > ${D}${libdir}/sota/conf.d/40-hardware-id.toml
+
```

Comment:
Originally I was thinking it would be best if we did this echo in a separate recipe, like one of the ones in the `recipes-sota/config` directory, but actually, we could just keep it here. But if we keep it here, it should be guarded to check that `HARDWARE_ID` is set and nonempty.